internal/imapwire/decoder.go

```diff
@@ -144,15 +148,27 @@ func (dec *Decoder) SP() bool {
 		return false
 	}
 	dec.mustUnreadByte()
+	// sure not end of line
 	return b == '('
 }
 
+// endOfLine returns true if the next bytes are a CR, LF or EOF.
```

Comment:
The comment seems out of sync with the implementation here? AFAIU false is returned on EOF?